src/handler.js

```diff
@@ -276,6 +277,22 @@ const sentMetrics = {
   }
 }
 
+const IpldCodecs = { 85: 'raw', 112: 'dag-pb', 113: 'dag-cbor' }
+const MultihashCodecs = { 18: 'sha256', 45600: 'blake2b256' }
+const HashSizes = { 256: '256' }
```

Comment:
do we care about hash size?

Reply:
Not loads, but it might be interesting if we find lots of something other than 256.